politeiawww/cmd/politeiavoter/politeiavoter.go

```diff
@@ -336,14 +338,14 @@ func (c *ctx) makeRequest(method, route string, b interface{}) ([]byte, error) {
 		}
 	}
 
-	fullRoute := c.cfg.PoliteiaWWW + v1.PoliteiaWWWAPIRoute + route +
-		queryParams
+	fullRoute := c.cfg.PoliteiaWWW + route + queryParams
```

Comment:
I think we should have the api (ex. `tkv1.APIRoute`) and the route (ex. `tkv1.RouteInventory`) passed in as separate arguments.  Combining them into a single argument makes it ambiguous what the `route` argument entails.

Reply:
agreed